main.go

```diff
@@ -214,6 +215,10 @@ func main() {
 			"All errors are tried using an exponential backoff strategy. "+
 			"The value must be greater than zero. "+
 			"Also set from environment variable VSO_BACKOFF_MULTIPLIER.")
+	flag.DurationVar(&cleanupOrphanedShadowSecretInterval, "cleanup-orphaned-shadow-secret-interval", 1*time.Hour,
+		"The time interval between each execution of the cleanup process for cached shadow secrets"+
+			"associated with a deleted HCPVaultSecretsApp. "+
+			"Also set from environment variable VSO_HVSA_CLEANUP_ORPHANED_SHADOW_SECRET_INTERVAL.")
```

Comment:
I wonder if we should add `hvsa` to the end of this to match the other flags? That seems to be the convention the other flags are following (like `min-refresh-after-hvsa`).

It's getting pretty long, so I think we could drop "orphaned" without losing too much meaning:

```suggestion
	flag.DurationVar(&cleanupOrphanedShadowSecretInterval, "cleanup-shadow-secret-interval-hvsa", 1*time.Hour,
		"The time interval between each execution of the cleanup process for cached shadow secrets"+
			"associated with a deleted HCPVaultSecretsApp. "+
			"Also set from environment variable VSO_CLEANUP_SHADOW_SECRET_INTERVAL_HVSA.")
```